components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/gen/java/org/wso2/carbon/apimgt/rest/api/store/WorkflowsApiService.java

```diff
@@ -1,31 +1,25 @@
-/*
- *
- *   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
- *
- *   WSO2 Inc. licenses this file to you under the Apache License,
- *   Version 2.0 (the "License"); you may not use this file except
- *   in compliance with the License.
- *   You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing,
- *  software distributed under the License is distributed on an
- *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- *  KIND, either express or implied.  See the License for the
- *  specific language governing permissions and limitations
- *  under the License.
- *
- */
 package org.wso2.carbon.apimgt.rest.api.store;
 
+import org.wso2.carbon.apimgt.rest.api.store.*;
+import org.wso2.carbon.apimgt.rest.api.store.dto.*;
```

Comment:
These are generated code. We do not edit these codes once generated. manually edited data will be loss when code is re generated again